src/schemas/markupmapping.description.md

```diff
@@ -0,0 +1,9 @@
+A markup mapping defines how Helix should generate markup for certain Markdown or DOM patterns.
+
+A configuration consits of a mandatory `match` expression, which is a CSS selector that operates either on the Markdown or DOM.
+
+Furthermore, a configuration can have any number of actions (including none at all), for example:
+
+- `wrap`: adds more HTML elements around the generated HTML
```

Comment:
Missing doc for `type`, and `inject`/`add` seem to be replaced by `class`/`attribute`